src/components/Navbar/Sidebar.js

```diff
@@ -22,3 +23,22 @@ export const SidebarData = [
     className: "nav-text",
   },
 ];
+
+function Sidebar() {
+  return (
+    <div>
+      {SidebarData.map((item, index) => {
```

Comment:
So, we probably don't need to capitalize the sidebar data -- reserve capitalizations for Component declarations